core/src/main/java/org/opensearch/sql/expression/function/BuiltinFunctionName.java

```diff
@@ -365,6 +370,8 @@ public enum BuiltinFunctionName {
           .put("stddev_samp", BuiltinFunctionName.STDDEV_SAMP)
           // .put("earliest", BuiltinFunctionName.EARLIEST)
           // .put("latest", BuiltinFunctionName.LATEST)
+          .put("list", BuiltinFunctionName.LIST)
+          .put("values", BuiltinFunctionName.VALUES)
```

Comment:
Is this required? Because I don't see we enable both in grammar and no test cover this (probably need to test with `eventstats`)?